frontend/src/components/SurveyPage/QuestionSegment.jsx

```diff
@@ -12,7 +12,7 @@ const QuestionSegment = ({ question, disabled, ...props }) => {
       case 0:
         return "red-outline";
       case 1:
-        return "yellow-outline";
+        return "red-outline";
       default:
         return "";
     }
```

Comment:
Instead of this function, you could install `classnames` library, then use it in `Textarea`:
```jsx
<TextArea className={classnames({'red-outline': !question.comment && question.answer < 2})} ... />
```

Reply:
I've changed it for ternary operator